### PR TITLE
build: upgrade phoenix and liveview

### PR DESCRIPTION
**Pull request by joaodiaslobo**
Update Phoenix to `1.7.0` and Phoenix Live View to `0.19.5`. Also replaces the icons package with another that only uses Heroicons.

### CODE REVIEW

File: lib/atomic_web.ex [hunk 1]
```diff
@@ -102,7 +103,7 @@ defmodule AtomicWeb do
 
       alias Atomic.Uploaders
       alias AtomicWeb.Router.Helpers, as: Routes
-      alias Icons.Heroicons
+      alias Heroicons
```
Comment (nelsonmestevao) [inline lib/atomic_web.ex]: No need to alias something to itself 👀
```suggestion
```

Comment (joaodiaslobo) [reply lib/atomic_web.ex]: That's true 😅, thanks for the heads up!

File: lib/atomic_web/live/hooks.ex [hunk 2]
```diff
@@ -3,6 +3,7 @@ defmodule AtomicWeb.Hooks do
   Ensures common `assigns` are applied to all LiveViews attaching this hook.
   """
   import Phoenix.LiveView
+  import Phoenix.Component
```
Comment (ruilopesm) [inline lib/atomic_web/live/hooks.ex:6]: Why is this import needed here?

Comment (joaodiaslobo) [reply lib/atomic_web/live/hooks.ex:6]: After Phoenix Live View 0.18.0 update, `assign/3` _(function used in this file)_ moved to `Phoenix.Component`, this move happened to multiple functions.

> You must import Phoenix.Component where you previously imported Phoenix.LiveView.Helpers when upgrading. You may also need to import Phoenix.Component where you also imported Phoenix.LiveView and some of its functions have been moved to Phoenix.Component.

[Read about it here](https://github.com/phoenixframework/phoenix_live_view/blob/main/CHANGELOG.md#0180-2022-09-20)

**Review by ruilopesm — COMMENTED** [review]
As per the documentation says, we should use the `link/1` function, provided by `Phoenix.Component`.
image

**Review by ruilopesm — APPROVED** [review]
